src/test/java/com/softserveinc/ita/alukavenko/BaseTest.java

```diff
@@ -0,0 +1,33 @@
+package com.softserveinc.ita.alukavenko;
+
+import com.softserveinc.ita.pageobjects.alukavenko.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class BaseTest {
```

Comment:
TestRunner

move to utils package

Reply:
Done, but isn't it strange that tests classed extend TestRunner class?
Sounds a little bit confusing to me.

Also I wasn't sure where to create utils package - either in main branch or in test. 
Utils sounds like something that is common for the project, not tests only. But in that case we need at least to change the dependecies, in order to testng was availible not only in the test scope. So for now I decided to create utils package in the test branch.